.github/workflows/github-projects.yml

```diff
@@ -15,5 +15,5 @@ jobs:
       - uses: actions/add-to-project@main
         with:
           project-url: https://github.com/orgs/openfoodfacts/projects/12
-          github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why did you change that @aadarsh-ram ?

And why did @teolemon us a PAT (I imagine it is because this action can trigger other actions ?) 
see https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow (in this case we can add a comment to make it clearer)

Reply:
Yep, so it for some reason failed for my first commit.

Reply:
So, since the GITHUB_TOKEN already has the required permissions of ADD_TO_PROJECT_PAT, I used that.
But, for my second commit, the action didn't run. Hence, I have no idea if it worked or not.

Reply:
The problem we have is that we hit the limit of github projects, it's not related to the token.

In fact I think that we should remove this action for now @teolemon ? (or maybe remove old enough bug from the project to gain space in the project ?)

Reply:
@alexgarel @stephanegigandet @teolemon This was the failed workflow run: https://github.com/openfoodfacts/openfoodfacts-server/actions/runs/2171303073
Since the error was related to a token, I wanted to see if it would work for the GITHUB_TOKEN.

Reply:
I archived fixed issues, I added all new issues, and we're below the threshold.

Reply:
For some reason, I'm not able to recreate the workflow error, since the issue has already been added to the project I think. 
So, I brought things back to how it was.